presto-hive/src/main/java/io/prestosql/plugin/hive/azure/PrestoAzureConfigurationInitializer.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.azure;
+
+import io.prestosql.plugin.hive.ConfigurationInitializer;
+import org.apache.hadoop.conf.Configuration;
+
+import javax.inject.Inject;
+
+import static java.lang.String.format;
+
+public class PrestoAzureConfigurationInitializer
+        implements ConfigurationInitializer
+{
+    private final String wasbAccessKey;
+    private final String wasbStorageAccount;
+
+    @Inject
+    public PrestoAzureConfigurationInitializer(HiveAzureConfig hiveAzureConfig)
+    {
+        this.wasbAccessKey = hiveAzureConfig.getWasbAccessKey();
+        this.wasbStorageAccount = hiveAzureConfig.getWasbStorageAccount();
+    }
+
+    @Override
+    public void initializeConfiguration(Configuration config)
+    {
+        if (wasbAccessKey != null && wasbStorageAccount != null) {
```

Comment:
we should raise an exception when only one of them is present (maybe in`HiveAzureConfig` in some `@PostConstruct` method)